Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,45 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                Console.Write("What is your name: ");
+                var name = Console.ReadLine();
+
+                Console.Write("What is your surname: ");
+                var surnname = Console.ReadLine();
+
+                Console.Write("What is your age: ");
+                var age = int.Parse(Console.ReadLine());
+
+                Console.Write("What is your weight in kilogram: ");
```

Comment:
I think mixing different styles is a very bad habit that should be adressed early on.
IF you decide on using var, you should use it in the same scenario in the same fashion.
This means for basic built-in types like string int float etc. you should either use var or use that type explicitely.
IF you break this rule you should have a good reason to do so because if other people have to read your code they might wonder about why you did this and this will confuse them if there is no valid reason behind this.

Reply:
In a later commmit ,  i already changed it this way and in a later commit, I changed it to ` Console.WriteLine($"Now for another person {Environment.NewLine}  ");  ` so it works also on a linux machine which uses `/r /n`  as far as I know. So I think this one is already resolved.

Reply:
Changed everythihg to var and I hope you are satisfied of the last commit